captum/attr/_core/noise_tunnel.py

```diff
@@ -43,13 +45,13 @@ def __init__(self, attribution_method):
 
     def attribute(
         self,
-        inputs,
-        nt_type="smoothgrad",
-        n_samples=5,
-        stdevs=1.0,
-        draw_baseline_from_distrib=False,
-        **kwargs
-    ):
+        inputs: Union[Tensor, Tuple[Tensor, ...]],
+        nt_type: str = "smoothgrad",
+        n_samples: int = 5,
+        stdevs: Union[float, Tuple[float, ...]] = 1.0,
+        draw_baseline_from_distrib: bool = False,
+        **kwargs: Any
+    ) -> Union[Tensor, Tuple[Tensor, ...]]:
```

Comment:
I realized this return type will have some issues with other methods, so we will look into those later, you can just remove the return type hint here.